sormas-backend/src/test/java/de/symeda/sormas/backend/labmessage/LabMessageFacadeEjbTest.java

```diff
@@ -1,166 +1,36 @@
-package de.symeda.sormas.backend.labmessage;
```

Comment:
I've splitted it because 
1) it was just a Unit test and my test case didn't work because everything was mocked
2) In sormas backed module we have only integration tests, and because we have only a few tests it's more important to have integration tests, so I created integration test